frontend/util.lua

```diff
@@ -284,4 +284,49 @@ function util.getMenuText(item)
     return text
 end
 
+function util.translateDayOfWeek(day_of_week)
+    local _ = require("gettext")
+    if day_of_week == "Monday" then return _("Monday")
+        elseif day_of_week == "Tuesday" then return _("Tuesday")
+        elseif day_of_week == "Wednesday" then return _("Wednesday")
+        elseif day_of_week == "Thursday" then return _("Thursday")
+        elseif day_of_week == "Friday" then return _("Friday")
+        elseif day_of_week == "Saturday" then return _("Saturday")
+        elseif day_of_week == "Sunday" then return _("Sunday")
+    end
+    return
+end
+
+function util.translateShortDayOfWeek(day_of_week)
+    local _ = require("gettext")
+    if day_of_week == "Mon" then return _("Mon")
+    elseif day_of_week == "Tue" then return _("Tue")
+    elseif day_of_week == "Wed" then return _("Wed")
+    elseif day_of_week == "Thu" then return _("Thu")
+    elseif day_of_week == "Fri" then return _("Fri")
+    elseif day_of_week == "Sat" then return _("Sat")
+    elseif day_of_week == "Sun" then return _("Sun")
+    end
+    return
+end
+
+
+function util.translateMonth(month)
```

Comment:
Two suggestions:

1. It's more efficient to implement these lookup using lookup tables. For example dayOfWeek translation can be done as:
```lua
dayOfWeekTranslation = {
    "Monday" = _("Monday"),
    ...
}
```

2. Don't put code into shared module unless it's really shared. In this case, those lookup tables are only used by statistics plugins. So it's better to put them inside the plugin directory. We can always move them back to the global util module if we ever need to reuse them in the future. But until then, keeping the change self contained will save us maintenance work.